src/Graphics/Texture.cs

```diff
@@ -290,6 +300,12 @@ SurfaceFormat format
 			{
 				switch (formatFourCC)
 				{
+					case 0x71:
```

Comment:
Need FOURCC constants for these two

Reply:
There are no FOURCC constants for these. These are D3DFORMAT enums. See the following MS code:

https://docs.microsoft.com/en-us/windows/uwp/gaming/complete-code-for-ddstextureloader

Reply:
Got it - let's comment with what the magic numbers mean, then this is good to go.

Reply:
Done!